contrib/compare_pack.py

```diff
@@ -0,0 +1,133 @@
+from pathlib import Path
+import shutil
+import click
+from nomenklatura.resolver import Linker
+from nomenklatura.store.base import View as BaseView
+from tempfile import mkdtemp
+from followthemoney.cli.util import InPath
+import json
+
+from zavod.archive import StatementGen, _read_fh_statements
+from zavod.dedupe import get_dataset_resolver
+from zavod.logs import get_logger
+from zavod.entity import Entity
+from zavod.meta import Dataset, get_catalog
+from zavod.store import Store as BaseStore
+
+log = get_logger(__name__)
+View = BaseView[Dataset, Entity]
+Dir = click.Path(dir_okay=True, readable=True, path_type=Path, allow_dash=True)
+
+
+class Store(BaseStore):
+    def __init__(
+        self,
+        name: str,
+        path: Path,
+        statements_path: Path,
+    ):
+        metadata = {
+            "name": name,
+            "title": name,
+            "summary": "Synthetic, ad-hoc virtual collection for testing ladi da",
+            "hidden": True,
+        }
+        catalog = get_catalog()
+        dataset = Dataset(catalog, metadata)
+        resolver = get_dataset_resolver(dataset)
+        linker: Linker[Entity] = resolver.get_linker()
+        super().__init__(dataset, linker, path)
+        self.entity_class = Entity
+        self.statements_path = statements_path
+
+    def iter_dataset_statements(self, external: bool = False) -> StatementGen:
+        with open(self.statements_path, "r") as fh:
+            yield from _read_fh_statements(fh, external)
+        return
+
+    def build(self, external: bool = False) -> None:
+        log.info("Building local LevelDB aggregator...", scope=self.dataset.name)
+        idx = 0
+        with self.writer() as writer:
+            stmts = self.iter_dataset_statements(external=external)
+            for idx, stmt in enumerate(stmts):
+                if idx > 0 and idx % 100000 == 0:
+                    log.info(
+                        "Indexing aggregator...",
+                        statements=idx,
+                        scope=self.dataset.name,
+                    )
+                stmt.dataset = self.dataset.name
+                writer.add_statement(stmt)
+        log.info("Local cache complete.", scope=self.dataset.name, statements=idx)
+
+
+def dump_dict(filename, dict_):
+    with open(filename, "w") as f:
+        for entity in dict_.values():
+            f.write(json.dumps(entity.to_dict()) + "\n")
+
+
+@click.option("--clear", is_flag=True, type=bool, default=False)
+@click.argument("working_dir", type=Dir)
+@click.argument("b_path", type=Dir)
+@click.argument("a_path", type=Dir)
+@click.command()
+def main(a_path: Path, b_path: Path, working_dir: Path, clear: bool = False):
```

Comment:
I'm sure we don't want contrib to just be a big dump of every random script. Where do you stuff scripts you use to verify stuff? I have a scripts branch, but maybe leaving that lying around on github is a bit messy.